snap/snapcraft.yaml

```diff
@@ -220,73 +194,11 @@ parts:
       'fastdds_no_shared_memory.xml': usr/share/
 
   cleanup:
-    after: [gazebo, ros2-foxy-extension]
+    after: [gazebo, ros2-humble]
```

Comment:
@Guillaumebeuzeboc  this passed with ros2-humble-extension on the first run, subsequent runs failed unless using ros2-humble as it is here. do we have an issue in our extension naming?

Reply:
any green run before the change above, for example https://github.com/canonical/gazebo_snap/actions/runs/6025428362/job/16346109543 yesterday

Reply:
The commit ref: d3e0df9f4c14218699a25a91a97036198525364b cannot be found in your branch. Where can I find this commit?

Reply:
So the comment show the commit ^^
and you are using the `ros2-humble` extension not the `ros2-humble-extension` extension
https://github.com/canonical/gazebo_snap/commit/d3e0df9f4c14218699a25a91a97036198525364b#diff-56759910381a014fecfd7556dd72ddd68c747d922a5b7df2044b9ce7c552f5f5R246

Reply:
Note that this lists other parts for which the `cleanup` must wait. It is not related to the extension name. Now the extension `ros-humble` do create its own part to dump the command-chain scripts. I don't recall the exact name of that part at the moment but I think you can omit it here.

Reply:
Unfortunately, in case of a typo snapcraft doesn't raise an error: https://bugs.launchpad.net/snapcraft/+bug/2033882
If the after keyword with the exact extension part name is not mentioned, there is no guarantee that the clean-up will happen after everything has been moved to `prime/`

Reply:
You mean when they write their own snap?